Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -798,14 +798,34 @@ def perform_rasterize(path: str | list[str],
     :param width: window width
     :param height: window height
     """
-    demisto.debug(f"perform_rasterize, {path=}, {rasterize_type=}")
+
+    # convert the paths param to list in case we have only one string
+    paths = argToList(paths)
+
+    # create a list with all the paths starts with "mailto:"
+    mailto_paths = [path_value for path_value in paths if path_value.startswith('mailto:')]
+
+    if mailto_paths:
+        # remove the mailto from path param
+        paths = list(set(paths) - set(mailto_paths))
+        demisto.error(f'perform_rasterize skip the following paths: {mailto_paths}')
+        return_results(CommandResults(
+            readable_output=f'URLs that start with "mailto:" cannot be rasterized.\nURL: {mailto_paths}'))
+
+    if not paths:
+        message = 'There are no valid paths to rasterize'
+        demisto.error(message)
+        return_error(message)
+        return None
+
+    demisto.debug(f"perform_rasterize, {paths=}, {rasterize_type=}")
     browser, chrome_port = chrome_manager()
 
     if browser:
         support_multithreading()
         with ThreadPoolExecutor(max_workers=MAX_CHROME_TABS_COUNT) as executor:
-            demisto.debug(f'path type is: {type(path)}')
-            paths = [path] if isinstance(path, str) else path
+            demisto.debug(f'path type is: {type(paths)}')
+            paths = [path] if isinstance(path, str) else path  # type: ignore
```

Comment:
not needed since
paths = argToList(paths)

Reply:
done